### DISCUSSION TITLE
Some challenges using pybind11 in Emscripten CMake builds

### DISCUSSION THREAD

**Discussion by nyue** [pybind/pybind11#5186] [Q&A]
I built and install pybind11 v2.12.0 with Emscripten with no issue

Now when I tried to use those heads in a pybind11 project, CMake is having issue with the pybind11Config.cmake file

From the documentation, it appears the Emscripten build has been working is 2.11.0

Any idea why I am getting the following when I try to use find_package on pybind11 ?

I saw some post mentioned about setting the ABI and tried that
```
set (Python3_FIND_ABI "ANY" "ANY" "ANY")
```
but it did not resolve the problem I am encountering.

```sh
CMake Error at CMakeLists.txt:5 (find_package):
  Could not find a configuration file for package "pybind11" that is
  compatible with requested version "".

  The following configuration files were considered but not accepted:

    /home/nyue/temp/whiskey-dependencies/wasm/share/cmake/pybind11/pybind11Config.cmake, version: 2.12.0 (64bit)
```

**Comment by TomExMachina** [comment]
I'm considering attempting to run in pyodide and wondering if you have any further findings/resolutions

**Reply by nyue** [reply]
I have put the WASM aspect of that project on hold until 2026, I have no additional findings to share.
